views/idea.handlebars

```diff
@@ -48,8 +54,8 @@
         {{/if}}
     </div>
 </div>
-
-{{#unless (is_prop_in idea.interests_status user_id)}}
+{{!-- {{#if can_join}} --}}
```

Comment:
Do we want to uncomment this part so that it hides the lower "Become a collaborator" section here too?

Reply:
Thanks for the catch!